lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -6638,6 +6638,10 @@ void EmitDestructuredArray(
 
 void EmitNameInvoke(Js::RegSlot lhsLocation,
     Js::RegSlot objectLocation,
+    Js::RegSlot propIdArrLocation,
```

Comment:
>propIdArrLocation [](start = 16, length = 17)

where is this used in this function?

Reply:
oops, leftover from a previous implementation strategy

---
In reply to: [205218436](https://github.com/Microsoft/ChakraCore/pull/5498#discussion_r205218436) [](ancestors = 205218436)